scripts/main.js

```diff
@@ -197,32 +197,52 @@ const data = [
   },
 ];
 
-function search(arr) {
-  let matchingStrings = [];
-
-  const cities = [
-    'Berlin',
-    'Santa Cruz de Tenerife',
-    'Vysokie Tatry',
-    'Bali',
-    'Rotterdam',
-    'Ourika'
-  ];
-
-  let matchingStringsIndex = 0;
-
-  for (let i = 0; i < arr.length; i++) {
-    if (cities.includes(arr[i].city)) {
-      const str = `${arr[i].country}, ${arr[i].city}, ${arr[i].hotel}`;
-      matchingStrings[matchingStringsIndex] = str;
-      matchingStringsIndex++;
+function newArr (data) {
+  let strArr=[];
+
+  for (let i = 0; i < data.length; i++) {
```

Comment:
It's bad solution, becouse you hardcoded result. If array of data will change, your function will not working.
Also, in this case you can search only city. Netherless, your previous solutionn was better,  you need to change it a little bit, and add check if your `arr[i]`includes city, country, hotel. For this case you should use `||` operator. or concatenate  it in one string and check if this string include search parameter if yes, retur `a[i]`